src/OpenApi/Factory/OpenApiFactory.php

```diff
@@ -843,4 +873,61 @@ private function mergeParameter(Parameter $actual, Parameter $defined): Paramete
 
         return $actual;
     }
+
+    /**
+     * @param string[]              $errors
+     * @param array<string, string> $responseMimeTypes
+     */
+    private function addOperationErrors(Operation $operation, array $errors, array $responseMimeTypes, ResourceMetadataCollection $resourceMetadataCollection, Schema $schema, \ArrayObject $schemas): Operation
+    {
+        $existingResponses = null;
+        foreach ($errors as $error) {
+            if (!is_a($error, ProblemExceptionInterface::class, true)) {
+                $this->logger?->warning(\sprintf('The error class "%s" does not implement "%s". Did you forget a use statement?', $error, ProblemExceptionInterface::class));
```

Comment:
```suggestion
                $this->logger?->warning(\sprintf('The error class "%s" does not implement "%s". Did you forget a use statement?', $error, ProblemExceptionInterface::class));

                continue;
```

But shouldn't we throw an exception instead of logging a warning? This will made it easier to catch this kind of errors during development.